6_visualize/src/prep_gage_sites_fun.R

```diff
@@ -1,25 +1,13 @@
-prep_gage_sites_fun <- function(stage_data, gage_col_config, DateTime){
-  this_DateTime <- as.POSIXct(DateTime, tz = "UTC") # WARNING, IF WE EVER MOVE FROM UTC elsewhere, this will be fragile/bad.
-  this_stage_data <- filter(stage_data, dateTime == this_DateTime)
-
-  norm_data_sites <- filter(this_stage_data, stage_normalized < flood_stage_normalized)
-  flood_data_sites <- filter(this_stage_data, stage_normalized >= flood_stage_normalized)
-  missing_data_sites <- filter(this_stage_data, is.na(stage_normalized))
-
-  plot_fun <- function(){
-
-    plot(sf::st_geometry(norm_data_sites$geometry), add = TRUE,
-         pch = 21, bg = gage_col_config$gage_norm_col, col = NA, cex = 1.5)
-    plot(sf::st_geometry(flood_data_sites$geometry),
-         pch = 21, bg = gage_col_config$gage_norm_col, add = TRUE,
-         col = gage_col_config$gage_flood_col, lwd = 4, cex = 1.7)
-    plot(sf::st_geometry(missing_data_sites$geometry),
-         pch = 4, add = TRUE,
-         col = gage_col_config$gage_norm_col, lwd = 5, cex = 1.3)
-    plot(sf::st_geometry(missing_data_sites$geometry),
-         pch = 4, add = TRUE,
-         col = gage_col_config$gage_out_col, lwd = 4, cex = 1.3)
-
+prep_gage_sites_fun <- function(percentile_color_data_ind, sites_sp, gage_style, dateTime){
+  this_date <- as.POSIXct(dateTime, tz = "UTC") #watch timezones if we ever switch from daily data
+  #only plotting sites with color for now
+  percentile_color_data <- readRDS(as_data_file(percentile_color_data_ind))
+  this_date_colors <- filter(percentile_color_data, dateTime == this_date)
+  sites_sp@data <- left_join(sites_sp@data, this_date_colors, by = "site_no")
+  gage_style <- gage_style$gage_style
```

Comment:
I'm not really following how `gage_style$gage_style` is needed. It looks like you pass in the result of `viz_config[[I('gage_style')]]` to this function, so you should be able to just say `gage_style$pch`, etc. Am I missing something?

Reply:
I thought that too, but if you put a `browser()` here, its ending up with an extra list level of `gage_style`.  I'm not sure why.

Reply:
Huh, ok. Not the end of the world, but I followed the same pattern and didn't have that issue

Reply:
Hypothesis: David, could it be the case that the first time you modified 6_visualize.yml, you had
```yml
  gage_style_config:
    command: viz_config[I('gage_style')]
```
and then when you changed it to double brackets (`command: viz_config[[I('gage_style_config')]]`), you still were getting a length-one list with an element named `gage_style_config`? If that's the case, you should force a rebuild of `gage_style` with `scmake('gage_style_config', force=TRUE)`. It's a remake bug that remake doesn't notice when you switch between single and double brackets within a remake command.

Reply:
Result: I deleted the one line Lindsay called out above and am now able to build a GIF successfully with the expected plot formatting. I'll make that change in my PR I'm about to submit.

Reply:
thought there must be something fishy going on